Hearthstone Deck Tracker/Windows/DeckScreenshotWindow.xaml.cs

```diff
@@ -0,0 +1,187 @@
+				$"Images/Bars/{heroId}.png", UriKind.Relative)), new Rect(54, 0, 130, 34)));
+			DrawingGroup.Children.Add(new ImageDrawing(new BitmapImage(new Uri(
+				"Images/Themes/Bars/dark/fade.png", UriKind.Relative)), new Rect(0, 0, 183, 34)));
+
+			return new ImageBrush {
+				ImageSource = new DrawingImage(DrawingGroup),
+				AlignmentX = AlignmentX.Left,
+				Stretch = Stretch.UniformToFill
+			};
+		}
+
+		private string GetTagText(Deck deck)
+		{
+			List<string> predefined = new List<string>() {
+				"Midrange",
+				"Aggro",
+				"Control",
+				"Tempo",
+				"Combo"
+			};
+			if(deck.Tags.Count > 0)
+			{
+				foreach(var tag in predefined)
+				{
+					var found = deck.Tags.Find(
+						f => f.ToLowerInvariant().Replace("-", "") == tag.ToLowerInvariant());
+					if(found != null)
+						return found;
+				}
+			}
+			return deck.Class;
+		}
+
+		private string GetFormatText(Deck deck)
+		{
+			var allTags = deck.TagList.ToLowerInvariant();
+			if(deck.IsArenaDeck)
+				return "Arena";
+			else if(allTags.Contains("brawl"))
+				return "Brawl";
+			else if(allTags.Contains("adventure") || allTags.Contains("pve"))
+				return "Adventure";
+			else if(deck.StandardViable)
+				return "Standard";
+			else
+				return "Wild";
+		}
+
+		private void ShowFormatIcon(Deck deck)
+		{
+			RectIconStandard.Visibility = Collapsed;
+			RectIconWild.Visibility = Collapsed;
+			RectIconArena.Visibility = Collapsed;
+			RectIconBrawl.Visibility = Collapsed;
+			RectIconAdventure.Visibility = Collapsed;
+
+			var allTags = deck.TagList.ToLowerInvariant();
+			if(deck.IsArenaDeck)
+				RectIconArena.Visibility = Visible;
+			else if(allTags.Contains("brawl"))
+				RectIconBrawl.Visibility = Visible;
+			else if(allTags.Contains("adventure") || allTags.Contains("pve"))
+				RectIconAdventure.Visibility = Visible;
+			else if(deck.StandardViable)
+				RectIconStandard.Visibility = Visible;
+			else
+				RectIconWild.Visibility = Visible;
+		}
+
+		private int TotalDust(Deck deck)
+		{
+			List<string> nonCraftableSets = new List<string>() {
+				"Curse of Naxxramas",
+				"Blackrock Mountain",
+				"League of Explorers",
+				"Basic"
+			};
+			List<string> nonCraftableCards = new List<string>() {
+				"OG_280", "OG_281"
+			};
+
+			return deck.Cards
+				.Where(c => !nonCraftableSets.Contains(c.Set) && !nonCraftableCards.Contains(c.Id))
+				.Sum(c => c.DustCost * c.Count);
+		}
+
+		private string ClassToID(string klass)
+		{
+			switch(klass.ToLowerInvariant())
+			{
+				case "druid":
+					return "HERO_06";
+
+				case "hunter":
+					return "HERO_05";
+
+				case "mage":
+					return "HERO_08";
+
+				case "paladin":
+					return "HERO_04";
+
+				case "priest":
+					return "HERO_09";
+
+				case "rogue":
+					return "HERO_03";
+
+				case "shaman":
+					return "HERO_02";
+
+				case "warlock":
+					return "HERO_07";
+
+				case "warrior":
+				default:
+					return "HERO_01";
+			}
+		}
+
+		internal void Shutdown()
+		{
```

Comment:
Looks like this is leftover from me copying `PlayerWindow`, will get rid of it and just call `Close()` directly.